flow/util/distributed.py

```diff
@@ -714,8 +732,7 @@ def set_algorithm(experiment_name, config):
             minimize=True
         )
         algorithm = AxSearch(ax_client=ax_client,
-                             points_to_evaluate=best_params,
-                             max_concurrent=args.jobs)
```

Comment:
Why remove `max_concurrent`? Did you test to see if `args.jobs` is still respected after this change?

Reply:
max_concurrent in AxSearch has deprecated, so it causes an error. I checked that ConcurrentcyLimiter (https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/197062988266b7608cccc4380452c1e941844063/flow/util/distributed.py#L753-L754) works well for Ax search.